### DISCUSSION TITLE
Αλλαγή ώρας για την Τετάρτη

### DISCUSSION THREAD

**Discussion by lefterisan** [courses-ionio/hci#1610] [Announcements]
Καλησπέρα, θα ήθελα να σας ρωτήσω αν υπάρχει θέμα την Τετάρτη μου βάλατε παρουσία στις 5-6 γίνετε να έρθω στις 6-7 ή θα υπάρξει πρόβλημα; ΑΜ : Π2020068

**Comment by epidrome** [comment]
υπάρχει πρόβλημα μόνο αν κάποιος έρθει (απρόσκλητος) σε γκρουπ που έχει ήδη πολλά (15) άτομα, το γκρουπ 6-7 πάντως φαίνεται σχετικά άδειο
